src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,39 @@
+package domain;
+
+import java.util.List;
+
+public class Lotto {
+
+  private final List<Integer> numbers;
+
+  public Lotto(List<Integer> numbers) {
+    this.numbers = numbers;
+  }
+
+  public LottoMatch compareLotto(WinningLotto winningLotto) {
+    List<Integer> winningNumbers = winningLotto.getWinningNumbers();
+    int bonusNumber = winningLotto.getBonusNumber();
+    int winningCounter = 0;
+    boolean bonusChecker = false;
+
+    for (int number : numbers) {
+      winningCounter += checkWinningCounter(winningNumbers, number);
+    }
+    if (numbers.contains(bonusNumber)) {
+      bonusChecker = true;
+    }
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요. 😃 
```suggestion
    int winningCounter = (int) numbers.stream()
        .filter(winningNumbers::contains)
        .count();

    boolean bonusChecker = numbers.contains(bonusNumber);
```